### PR TITLE
go-selinux is failing to build on non linux systems

### PR DESCRIPTION
**Pull request by rhatdan**
This patch makes go-selinux at least compile on non-linux systems
Basically I stubbed out all of the public content on selinux.go
for non linux systems, and copied the original selinux.go to selinux_linux.go.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
@vbatts PTAL

**Comment by rhatdan** [comment]
@runcom @mrunalp PTAL

**Comment by baude** [comment]
LGTM, tested and it worked nicely

**Comment by runcom** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

File: go-selinux/selinux_linux.go [hunk 1]
```diff
@@ -0,0 +1,721 @@
+// +build linux
```
Comment (wking) [inline go-selinux/selinux_linux.go]: If the idea is to keep the dummy code in this repo, I think we need a more specific tag here as well.  Linux users may have a kernel that doesn't have SELinux compiled in, or they may prefer the stubs for a particular build for some other reason.

Comment (rhatdan) [reply go-selinux/selinux_linux.go]: SELinux code will just work without the selinux being compiled in on a linux system,  This is not an issue on non SELinux machines.

Comment (wking) [reply go-selinux/selinux_linux.go]: > SELinux code will just work without the selinux being compiled in on a linux system, This is not an issue on non SELinux machines.

I've filed #32 with a stub tag.  Can we continue this discussion there?

**Pull request merged**